dev.py

```diff
@@ -696,6 +696,7 @@ class Test(Task):
         help="more verbosity")
     # removed doctests as currently not supported by _lib/_testutils.py
     # doctests = Option(['--doctests'], default=False)
+    doctests = Option(['--doctests'], default=False, is_flag=True)
```

Comment:
As long as you have a special `doctest` command, this should not be needed.

Reply:
I wasn't sure about completely removing the option of running doctests using the `python dev.py test --doctests` incantation. I think now we have settled on: `python dev.py doctest`?

Reply:
I personally on a fence on which of the two way makes it. Not the two though.